js/src/dapps/registry/events/reducers.js

```diff
@@ -0,0 +1,24 @@
+const initialState = [];
+
+const sortEvents = (a, b) => {
+  const d = a.block.minus(b.block).toFixed(0);
+  if (d === 0) return a.index.minus(b.index).toFixed(0);
+  return d;
```

Comment:
It describes function flow better:
1. Try `block`
2. If ok, return result
3. Try `index`

`if (d !== 0)` would also be ok with me, I don't like hiding important (main) logic of a function inside if statements - I prefer to early-exit in case of failure / quick success instead, hence the comment.